frontend/tests/controllers/QualityNominationTest.php

```diff
@@ -19,6 +19,20 @@ public function qualityNominationsDemotionStatusProvider(): array {
             ['warning', \OmegaUp\ProblemParams::VISIBILITY_PUBLIC_WARNING],
         ];
     }
+
+    /**
+     * A PHPUnit data provider for all the tests that can accept a column for search nominations.
+     *
+     * @return list<list<mixed>>
+     */
+    public function qualityNominationsDemotionSearchColumnsProvider(): array {
+        return [
+            ['alias', 1],
+            ['author_username', 1],
+            ['nominator_username', 1]
```

Comment:
```suggestion
            ['nominator_username', 1],
            ['invalid column', 0],
```

con este cambio, `testSearchNominationsWrongQuery()` se puede eliminar!

Reply:
@ si se podria agregar la column invalida al provider ya que apiList al mandarle una columna que no es de las permitidas arroja una excepcion?

Reply:
oh buen punto. entonces se puede cambiar en vez de `invalid column` a que sea `nominator_username` con un valor que sabemos que no existe (como `"invalid username"`)?

Reply:
listo